pom.xml

```diff
@@ -12,7 +12,7 @@
   <properties>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <surefire-plugin.version>2.22.0</surefire-plugin.version>
-    <quarkus.version>0.19.1</quarkus.version>
+    <quarkus.version>1.0.0.Final</quarkus.version>
```

Comment:
❤️

Reply:
It built and passed so that means it's ok am-i-right?

Reply:
#worksforme ?

Still needs Susie et al to do some verification testing to ensure we don't have regressions. I think it should be fine, but we should be sure in case...